tjreports/plugins/user/tjreportsindexer/tjreportsindexer.php

```diff
@@ -135,7 +135,14 @@ protected function addIndexerEntry($user)
 				continue;
 			}
 
-			$value = $fieldsDetails[$key]->rawvalue;
+			if ($fieldsDetails[$key]->type == 'list')
+			{
+				$value = $fieldsDetails[$key]->rawvalue[0];
```

Comment:
What'll happen in case of a multi-select field ?

cc @manojLondhe @praneettekdi

Reply:
Yes,  we can receive multiple values. Right now we can store comma-separated values that would have already been stored with the previous code.  @coolbung @manojLondhe Let me know if the approach is ok.
@shindebalu You said only one value can be selected but I have tested and we can select multiple values. Test all the field types properly and make the necessary changes.

Reply:
@manojLondhe sir,
make the required changes.